src/components/modules/DetailedPage/ProductHeader/ProductHeader.tsx

```diff
@@ -1,49 +1,63 @@
 import React from "react";
+import avatar from "../../../../images/avatar.png";
 import {
   HeaderBlock,
-  HeaderTitle,
-  HeaderSubtitle,
+  ColumnTitle,
+  Type,
   Username,
   Tags,
   Date,
   Column,
   Row,
   Commitment,
+  Vision,
+  UsernameAvatar,
 } from "./StyledProductHeader";
 
 interface ProductHeaderProps {
   title: string;
-  subtitle: string;
-  tags: string[];
+  vision: string;
+  type: boolean;
+  keywords: string[];
   date: string;
-  commitment: string;
+  commitmentLevel: string;
```

Comment:
I think should spliced in half, to minCommitmentHours and maxCommitmentHours and they should only accept numbers!

Reply:
So i was looking at Nacer's implementation of types in his component and I was wondering If i can just use that instead of defining my own types?

Reply:
@mfvil if you want Nacer's implementation you'd need to merge his branch to yours. plus his change I'm requesting is not included in his types since they're not updated.

Reply:
Nacer's branch is [nacer/OpportunitiesFilter](https://github.com/dev-launchers/Dev-Recruiters/tree/nacer/OpportunitiesFilter)

but as I'm mentioned it doesn't have this type you're dealing with!